GDE-55: Our customers infinite scroll.

steveworley opened the pull request:
- Updates the our customer view to use the views infinite scroll module.
- Updates stylesheets to apply correct styles to the views.scss.
- Adds a views-inifinite-scroll template to match the uikit.

steveworley committed:
GDE-55: Our customers infinite scroll.

- Updates the our customer view to use the views infinite scroll module.
- Updates stylesheets to apply correct styles to the views.scss.
- Adds a views-inifinite-scroll template to match the uikit.

alan-cole commented on docroot/themes/custom/govcms_uikit_starterkit/sass/layout/_views.scss:
```diff
@@ -4,6 +4,10 @@
 
 .view {
+  .views-infinite-scroll-content-wrapper {
+    display: flex;
```
All contact cards are currently displayed inline. Adding `flex-wrap: wrap;` should fix this.

steveworley committed:
Merge branch 'master' into feature/gde-55-our-customers-infinite-scroll

steveworley committed:
GDE-55: Fixes flex-inline for our customer list.

- Updates card styles to remove margins on first and 4th.
- Updates the flex definition to display inline.

alan-cole commented:
Mobile styles are not implemented, but assuming that's a new story.
